### ISSUE TITLE
Warning: bitwise operands

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6356]
Newer appleclang warn for:

```
13:44:52 BIOCONDA INFO (ERR) In file included from /Users/builder/mambaforge/envs/bioconda/conda-bld/openms-meta_1664364460460/work/src/openms/source/KERNEL/ChromatogramPeak.cpp:36:
13:44:52 BIOCONDA INFO (ERR) In file included from /Users/builder/mambaforge/envs/bioconda/conda-bld/openms-meta_1664364460460/work/src/openms/include/OpenMS/KERNEL/MSSpectrum.h:39:
13:44:52 BIOCONDA INFO (ERR) /Users/builder/mambaforge/envs/bioconda/conda-bld/openms-meta_1664364460460/work/src/openms/include/OpenMS/KERNEL/RangeManager.h:108:14: warning: use of bitwise '&' with boolean operands [-Wbitwise-instead-of-logical]
13:44:52 BIOCONDA INFO (ERR)       return contains(inner_range.min_) & contains(inner_range.max_);
13:44:52 BIOCONDA INFO (ERR)              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
13:44:52 BIOCONDA INFO (ERR)                                         &&
13:44:52 BIOCONDA INFO (ERR) /Users/builder/mambaforge/envs/bioconda/conda-bld/openms-meta_1664364460460/work/src/openms/include/OpenMS/KERNEL/RangeManager.h:108:14: note: cast one or both operands to int to silence this warning
13:44:53 BIOCONDA INFO (ERR) 1 warning generated.
```

**Comment by cbielow** [comment]
I will look into it

**Comment by cbielow** [comment]
fixed in #6620
